tests/utilities/test_all_gather_grad.py

```diff
@@ -95,3 +95,26 @@ def training_epoch_end(self, outputs) -> None:
 
     trainer.fit(model)
     assert model.training_epoch_end_called
+
+
+@RunIf(min_gpus=2, skip_windows=True, special=True)
```

Comment:
this won't run
we need to add add it to `tests/special_tests.sh`
`special=True` means this test gets skipped by default
and `skip_windows` would be redundant in that case.

Reply:
Yeah, I noticed this, I actually commented out `special=True` when I test this. I put it here just because I was following the same settings of other tests in this script.